front/package.json

```diff
@@ -62,19 +56,15 @@
     "date-fns": "^3.6.0",
     "dd-trace": "^5.17.0",
     "embla-carousel-react": "^8.0.1",
-    "emoji-mart": "^5.5.2",
-    "event-source-polyfill": "^1.0.31",
     "eventsource-parser": "^1.0.0",
     "fast-diff": "^1.3.0",
     "formidable": "^3.5.1",
     "fp-ts": "^2.16.5",
     "fs-extra": "^11.1.1",
     "googleapis": "^118.0.0",
-    "googleapis-common": "^6.0.4",
     "hot-shots": "^10.0.0",
     "io-ts": "^2.2.20",
     "io-ts-reporters": "^2.0.1",
-    "io-ts-types": "^0.5.19",
```

Comment:
Likely the culrpit @Fraggle?

Reply:
io-ts-types is not imported in the code except in dust-tt/types, so I'm puzzled.
Trying to repro/config.